packages/page-accounts/src/modals/IdentityMain.tsx

```diff
@@ -100,8 +103,8 @@ function IdentityMain ({ address, className = '', onClose }: Props): React.React
 
       setData(info.web, setHasWeb, setValWeb);
 
-      [info.display, info.email, info.legal, info.riot, info.twitter, infoDiscord, info.web].some((info: Data) => {
-        if (info.isRaw) {
+      [info.display, info.email, info.legal, info.riot, info.twitter, infoDiscord, info.web].some((info: Data | null) => {
```

Comment:
See new `IdentityInfo`: https://github.com/polkadot-fellows/runtimes/blob/v1.2.1/system-parachains/people/people-kusama/src/people.rs#L96-L143

- `riot` is now `matrix`
- `github` and `discord` are first class members
- `additional` is no more

```suggestion
      [info.display, info.email, info.legal, info.matrix, info.twitter, info.discord, info.github, info.web].some((info: Data | null) => {
```

Reply:
Anyone with `additional` fields of `github` or `discord` had those fields migrated into the new struct.

Reply:
Ahh thanks for linking that, I will add those additions - Also need to keep the old structures though to make sure other chains don't have this break in the process.